src/core/toga/app.py

```diff
@@ -16,6 +16,14 @@ class MainWindow(Window):
     def __init__(self, id=None, title=None, position=(100, 100), size=(640, 480), factory=None):
         super().__init__(id=id, title=title, position=position, size=size, factory=factory)
 
+class WindowsRegistry(set):
```

Comment:
`WindowsRegistry` is a bit of a loaded term - there's an OS level service with this name on MS Windows. Something more direct like `WindowSet` would be better.

Reply:
Fair point. I've renamed to WindowSet in 5b08573e8.